modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -80,6 +84,11 @@ def set_pending_state_on_claim(auto_claim)
       save_auto_claim!(auto_claim, ClaimsApi::AutoEstablishedClaim::PENDING)
     end
 
+    def set_errorede_state_on_poa(poa)
```

Comment:
sp

Reply:
[af7d20b](https://github.com/department-of-veterans-affairs/vets-api/pull/19165/commits/af7d20bcd47c1631c3d8110347a0ba329ecfd2d7)